docs/tutorials/local_iteration_series/testing_with_artificial_conditions.md

```diff
@@ -3,16 +3,19 @@ id: testing_with_artificial_conditions
 title: Local Iteration - Testing multiplayer games with artificial network conditions
 description: Guide covering the available tools for local testing of multiplayer games with artificial latency, jitter and packet loss .
 ---
-- [General advice](#general-advice)
+- [How much lag/packet loss/jitter should we use?](#how-much-lagpacket-lossjitter-should-we-use)
 	- [Different network conditions for different peers](#different-network-conditions-for-different-peers)
 - [Unity Transport - Simulator Tools](#unity-transport---simulator-tools)
 	- [Clone-based workflow (ParrelSync)](#clone-based-workflow-parrelsync)
 	- [Debug Builds](#debug-builds)
 - [System-wide network conditioners](#system-wide-network-conditioners)
-	- [Clumsy on Windows](#clumsy-on-windows)
+	- [Clumsy (Windows)](#clumsy-windows)
 		- [Settings quickstart](#settings-quickstart)
-	- [Network Link Conditioner on OSX](#network-link-conditioner-on-osx)
+	- [Network Link Conditioner (Mac OS)](#network-link-conditioner-mac-os)
```

Comment:
nit: I think apple now writes it with a different capitalization (macOS) https://www.apple.com/ca/macos/big-sur/